plugins/events-backend/src/service/InMemoryEventBroker.ts

```diff
@@ -25,6 +25,8 @@ import { Logger } from 'winston';
  * In-memory event broker which will pass the event to all registered subscribers
  * interested in it.
  * Events will not be persisted in any form.
+ *
+ * @public
  */
 // TODO(pjungermann): add prom metrics? (see plugins/catalog-backend/src/util/metrics.ts, etc.)
 export class InMemoryEventBroker implements EventBroker {
```

Comment:
Now that this is becoming public API I think we scrutinize the name a bit. This doesn't really strike me as an "in-memory" thing, as it has no memory. All it does is to immediately forward published events to matching subscribers. I guess `InProcess*` or something like that could make sense, but I think we could probably keep it even more simple. Is this maybe a `DirectEventBroker`, `ImmediateEventBroker`, or `DefaultEventBroker`? If it wasn't taken already I might even have suggested we call this an `EventRouter` 😅

Reply:
I like DefaultEventBroker :)